index.js

```diff
@@ -1,6 +1,12 @@
 const zipCodes = require('./lib/zipCodes.js');
 const apiClient = require('./lib/apiClient.js');
 
+Params = {
+    zip : undefined,
+    lang : "en"
+};
```

Comment:
This makes the sheltersByZip function impure... let's convert this to just be two different variables.